Sources/Aardvark/Aardvark.swift

```diff
@@ -17,27 +17,38 @@
 import CoreAardvark
```

Comment:
Couple of small headerdoc tweaks in here, but most of the diff is adjusting line wrapping (now wraps at 120 characters).